python/cugraph/cugraph/sampling/uniform_neighbor_sample.py

```diff
@@ -342,13 +353,15 @@ def uniform_neighbor_sample(
         else None,
         h_fan_out=fanout_vals,
         with_replacement=with_replacement,
-        do_expensive_check=False,
+        do_expensive_check=True,
```

Comment:
was this intentional?

Reply:
not intentional, whoops.  We need to fix that.